### PR TITLE
updated code sample 

### PR DESCRIPTION
**Pull request by dom-inic**
In reference to issue #9443 updated the code sample to the new functions set_framework() and reduce_mean()

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -117,15 +117,15 @@ You can immediately use Ivy to train a neural network, using your favorite frame
             x = ivy.relu(self.linear0(x))
             return ivy.sigmoid(self.linear1(x))
 
-    ivy.set_backend('torch')  # change to any backend!
+    ivy.set_framework('torch')  # change to any framework!
```
Comment (hirwa-nshuti) [inline README.rst:120]: `set_framework` is used in the pypi version `Ivy v1.1.9` and as of now we are using the development version we use `set_backend`

File: README.rst [hunk 2]
```diff
@@ -117,15 +117,15 @@ You can immediately use Ivy to train a neural network, using your favorite frame
...
     model = MyModel()
     optimizer = ivy.Adam(1e-4)
     x_in = ivy.array([1., 2., 3.])
     target = ivy.array([0.])
 
     def loss_fn(v):
         out = model(x_in, v=v)
-        return ivy.mean((out - target)**2)
+        return ivy.reduce_mean((out - target)**2)
```
Comment (hirwa-nshuti) [inline README.rst:128]: Also, we are now using `ivy.mean`

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hi, thanks for contributing to ivy😊
Here are a few comments on your changes and an overview review is that you're likely using the pypi version which used these terms you're suggesting.

Thanks.

File: README.rst (hunk 1 above)
Comment (dom-inic) [reply README.rst:120]: Thanks for the review @hirwa-nshuti, i now understand thought there were inconsistencies in the documentation.

File: README.rst (hunk 2 above)
Comment (dom-inic) [reply README.rst:128]: okay

**Comment by hirwa-nshuti** [comment]
Yea there is some few inconsistency in documentation which we are working on now before the next release

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
I think these changes you suggested already exists

**Comment by dom-inic** [comment]
yeah they do, this PR can be closed for now

**Pull request closed without merging**